poc/dynamic-renderer.php

```diff
@@ -13,14 +13,13 @@ public function registerContentType($type, Callable $handler) {
 	public function __call($method, $arguments) {
 		if (preg_match('~^visit(.+)~', $method, $match)) {
 			$type = strtolower($match[1]);
-			if (!isset($this->handlers[$type])) {
-				$type = 'default';
-			}
```

Comment:
But why is there a need to set default. If the handler is undefined, the control straightaway goes to the else part and the statement is displayed.